src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/Queries/WhereInputObjectGraphType.cs

```diff
@@ -17,37 +28,37 @@ public override object ParseDictionary(IDictionary<string, object> value)
     }
 
     // Applies to all types.
-    public static readonly Dictionary<string, string> EqualityOperators = new()
+    public static readonly Dictionary<string, Func<IStringLocalizer, string, string>> EqualityOperators = new()
     {
-        { "", "is equal to" },
-        { "_not", "is not equal to" },
+        { "",  (S, description) => S["{0} is equal to", description] },
+        { "_not",  (S, description) => S["{0} is not equal to", description] },
     };
 
     // Applies to all types.
-    public static readonly Dictionary<string, string> MultiValueComparisonOperators = new()
+    public static readonly Dictionary<string, Func<IStringLocalizer, string, string>> MultiValueComparisonOperators = new()
     {
-        { "_in", "is in collection" },
-        { "_not_in", "is not in collection" },
+        { "_in", (S, description) => S["{0} is in collection", description] },
```

Comment:
I think here `description` should be `fieldName`, right? If so, we should rename it for better readability.

Reply:
It currently takes the description of the field. Maybe we should change that, but we can do that afterwards as it requires more changes.

Reply:
Description does take a description. However the call back accepts a stringLocalizer then a field name. From that field name it builds the localized string. `{ "_in", (S, description) => callback },`

I think `{ "_in", (S, description) => S["{0} is in collection", fieldName] },` should be `{ "_in", (S, description) => S["{0} is in collection", fieldName] },`

Reply:
That does not align with the current usage of the delegate and could be confusing. It takes the full description of the 'parent' field, not the name alone. E.g. "content item id is not equal to" instead of "contentItemId is not equal to".